sensor_msgs/Image image
---
object_identification/AttributeArray attributes
